src/C6 다이나믹 프로그래밍/Q32 JS2 정수 삼각형.js

```diff
@@ -0,0 +1,18 @@
+function solution(n, triangle) {
+  let pre = triangle[0];
+  let temp = [];
+
+  for (let i = 1; i < n; i++) {
+    temp = triangle[i].map((val, idx) => {
+      if (idx === 0) return pre[0] + val;
+      if (idx === i) return pre[i - 1] + val;
+      return Math.max(pre[idx - 1], pre[idx]) + val;
+    });
+    pre = [...temp];
```

Comment:
```suggestion
    pre = triangle[i].map((val, idx) => {
      if (idx === 0) return pre[0] + val;
      if (idx === i) return pre[i - 1] + val;
      return Math.max(pre[idx - 1], pre[idx]) + val;
    });
```

바로 `pre` 에 할당하셔도 좋지 않을까 싶습니다~

Reply:
임시변수를 줄이도록 더 신경써야겠네요 🧐